packages/unplugin-vue-i18n/src/index.ts

```diff
@@ -815,7 +817,8 @@ const mergeDeep = (target, ...sources) => {
 
 export default mergeDeep({},
   ${codes.map(code => `{${code}}`).join(',\n')}
-);`
+);
+export const SUPPORTED_LOCALES = ${JSON.stringify(locales)};`
```

Comment:
We prefer to support loaded locales with the API implemented in vue-i18n, not with constants exported in virtual modules.

vue-i18n keeps the locale internally. If the API is provided on the virtual module side, locales conflicts may occur.